lib/configuration/lib/consts.js

```diff
@@ -92,12 +92,12 @@ Settings : {{{displaySettings details.settings}}}
   DEFAULT_PR_VALIDATE: [{
     do: 'title',
     must_exclude: {
-      regex: 'wip|dnm|exp|poc'
+      regex: '^wip'
     }
   }, {
     do: 'label',
     must_exclude: {
-      regex: 'work in progress|do not merge|experimental|proof of concept'
+      regex: 'work in progress|wip|do not merge'
```

Comment:
I would throw `wip` here. Intentions fails with Issue title like `Creating an user wipes all others`. Would keep only verbose one, for demo purposes.

Reply:
\cc @jusx

Reply:
@rgtk - This is validating on the `label` and not the `title`. A label named `wip` seem reasonable here. Thoughts?

Reply:
Right. Thanks.